MapboxVisionAR/VisionARDelegate.swift

```diff
@@ -9,11 +9,11 @@
 import Foundation
 
 public protocol VisionARDelegate: class {
-    func onARCameraUpdated(visionARManager: VisionARManager, camera: ARCamera)
-    func onARLaneUpdated(visionARManager: VisionARManager, lane: ARLane?)
+    func visionARManager(visionARManager: VisionARManager, didUpdateARCamera camera: ARCamera)
```

Comment:
Was this really supposed?..

Reply:
This is kind a standard naming for delegate in iOS ecosystem

Reply:
I mean function renaming `onARCameraUpdated` -> `visionARManager`

Reply:
Discussed